doc/engineering-reference/src/simulation-models-encyclopedic-reference-004/radiant-system-models.tex

```diff
@@ -855,89 +855,85 @@ \subsubsection{Modeling Considerations/Assumptions}\label{modeling-consideration
 
-The ratio of the actual energy exchange to the maximum possible is known as the effectiveness of the heat exchanger, ε.  Thus, $ε = \frac{q}{q_{max}}$.
+The ratio of the actual energy exchange to the maximum possible is known as the effectiveness of the heat exchanger, $\varepsilon$
+
+$\varepsilon = \frac{q}{q_{max}}$.
 
 For the case when one fluid is stationary in a heat exchanger, Incropera and DeWitt (1985) state that the effectiveness of a heat exchanger can be related to the number of transfer units (NTU) through the following relation:
 
-$ε = 1 - e^{-NTU}$
+$\varepsilon = 1 - e^{-NTU}$
 
 and NTU is defined by:
 
 $NTU = \frac{UA}{ \PB{m c_p}_{water} }$
 
 The above equations will be used to model the performance of the panel within EnergyPlus and also to obtain a “design” UA value for the panel.
 
-Calculating the UA value of a cooling panel or coming up with a relationship for such a term is not a simple matter.  However, most devices are either rated or tested by the manufacturer under certain conditions.  From this rating information, a UA value can be calculated.  The EnergyPlus model will request rating information (inlet water temperature, mass flow rate, “air” temperature, and panel output under rating conditions) and then use it to calculate a UA value that will be used to model the performance of the panel.
+Calculating the UA value of a cooling panel or coming up with a relationship for such a term is not a simple matter.  However, most devices are either rated or tested by the manufacturer under certain conditions.  From this rating information, a UA value can be calculated.  The EnergyPlus model requests rating information (inlet water temperature, mass flow rate, “air” temperature, and panel output under rating conditions) and then uses it to calculate a UA value that is used to model the performance of the panel.
 
-The development of the method for estimating the panel UA value will use the same equations shown above but simply with the purpose of calculating UA rather than the performance of the device, i.e. determining the output of the panel.  The method for estimating the UA value of the panel begins with the panel rating data that includes the following values:
+The development of the method for estimating the panel UA value uses the same equations shown above but simply with the purpose of calculating UA rather than the performance of the device, i.e. determining the output of the panel.  The method for estimating the UA value of the panel begins with the panel rating data that includes the following values:
 
 \begin{itemize}
 \tightlist
 \item
-  Rating Inlet Water Temperature (Trwi)
+  Rating Inlet Water Temperature $\PB{T_{rwi}}$
```

Comment:
Just need to capture the math inside a math mode environment to fix the build errors here.

Reply:
@Myoldmopar Thanks, Edwin, for cleaning this up.  I'll try to remember to keep all things like that inside math mode.